composer.json

```diff
@@ -21,11 +21,11 @@
   },
   "require": {
     "php": "^5.6 || ^7.0",
-    "psr/http-message": "~1.0"
+    "psr/http-message": "^1.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.6 || ^5.5",
-    "zendframework/zend-coding-standard": "~1.0.0",
+    "phpunit/phpunit": "^5.6 || ^6.0",
+    "zendframework/zend-coding-standard": "^1.0",
```

Comment:
This is one of the few that needs to use the `~` operator, as we don't want new PRs to suddenly fail due to new rules being added to the standard.